streaming/src/test/java/org/apache/spark/streaming/JavaAPISuite.java

```diff
@@ -1244,7 +1244,15 @@ public void testCountByValue() {
     JavaTestUtils.attachTestOutputStream(counted);
     List<List<Tuple2<String, Long>>> result = JavaTestUtils.runStreams(ssc, 3, 3);
 
-    Assert.assertEquals(expected, result);
+
+
+    for (int i=0;i<expected.size();i++) {
+      List<Tuple2<String,Long>> expectedTupleList = expected.get(i);
+      List<Tuple2<String,Long>> resultTupleList = result.get(i);
+      Assert.assertTrue(resultTupleList.containsAll(expectedTupleList));
+    }
+
+    //Assert.assertEquals(expected, result);
```

Comment:
I think we might not need to leave this comment as it is able to be tracked down via blame button.